Sources/ComposableArchitecture/Store.swift

```diff
@@ -314,6 +314,23 @@ public final class Store<State, Action> {
     #endif
   }
 
+	/// A variant of ``scope(state:action:)-9iai9`` which hands the child state along with
+  /// the child action for scoping `action`.
+  @_disfavoredOverload
```

Comment:
I added this to guarantee the "normal" scope function is overwhelmingly preferred by the compiler and when using `CasePath`s. Not sure if it's actually needed but I don't think it harms it either.